ruby-training/trainees/alan/test_katas/03/JadenCase/spec/jaden_case_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rspec/autorun"
+require "../jaden_case.rb"
+
+describe JadenCase do
+
+    describe ".to_word_array" do
+        it "should return an array of each word in the sentence" do
+            expect(JadenCase.to_array_of_words("Hello, there.")).to eq(["Hello,","there."])
+        end
+    end
+
+    describe ".capitalize_words" do
```

Comment:
This is a good example of a method doing too much. The method is called `.capitalize_words`, but the return value you expect isn't only capitalized, it also splits the string into separate strings and puts them in an array. If I saw this method in a library and read the name, I'd assume it only capitalizes words I sent it and nothing else, which would result in my being unpleasantly surprised if used in production.